srcpkgs/snore/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'snore'
+pkgname=snore
+version=0.2
+revision=1
+build_style=gnu-makefile
+short_desc="Sleep with feedback"
+maintainer="wael <40663@protonmail.com>"
+license="MIT"
+homepage="https://github.com/clamiax/snore"
+distfiles="https://github.com/clamiax/$pkgname/archive/refs/tags/${version}.tar.gz"
+checksum=2b160061589e54b6a88b2995028491b38f9695393bf48af5ad61835fb6fd771a
+CFLAGS="-DVERSION=\\\"${version}\\\""
```

Comment:
are the quotes really needed here? i'd expect `-DVERSION=0.2` to work fine

Reply:
`CFLAGS="-DVERSION=${version}"` and `CFLAGS="-DVERSION=${version}"` will produce:
```
snore.c: In function 'main':
snore.c:112:15: error: expected ')' before numeric constant
  112 |   die("snore-"VERSION"\n");
      |               ^
      |               )
make: *** [Makefile:20: snore.o] Error 1
=> ERROR: snore-0.2_1: do_build: '${make_cmd} CC="$CC" CXX="$CXX" LD="$LD" AR="$AR" RANLIB="$RANLIB" CPP="$CPP" AS="$AS" OBJCOPY="$OBJCOPY" OBJDUMP="$OBJDUMP" CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" ${makejobs} ${make_build_args} ${make_build_target}' exited with 2
=> ERROR:   in do_build() at common/build-style/gnu-makefile.sh:9
```
i also took this from other templates.